scaleout/project/values.yaml

```diff
@@ -0,0 +1,18 @@
+
+
+
+
+labs:
+  image: jupyter/minimal-notebook:latest
+  repository: "git@github.com:scaleoutsystems/scaleout-alliance-orchestrator-sdk.git"
```

Comment:
scaleout-cli.git ?

Reply:
No this is testing to load a private repository with a custom key present. Much similar to the intended tennant/project instance (and key). Hence the orchestrator. (perhaps a bad example but working).